dashboard/app/controllers/levels_controller.rb

```diff
@@ -26,7 +26,7 @@ def show
     set_videos_and_callouts
     view_options(
         full_width: true,
-        no_footer: @game == Game.applab
+        no_footer: @game == Game.applab || @game == Game.netsim
```

Comment:
Could this maybe be a property on Game? e.g. around [here](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/game.rb#L76-76) add a `has_footer?`